README.md

```diff
@@ -1256,6 +1256,19 @@ officially supported LSP server for Ruff.
 However, Ruff is also available as part of the [coc-pyright](https://github.com/fannheyward/coc-pyright)
 extension for `coc.nvim`.
 
+<details>
+<summary>Ruff works with <a href="https://github.com/dense-analysis/ale"><code>ale</code></a>.</summary>
+<br>
```

Comment:
Why is this `<br>` necessary?

Reply:
Not sure, I just copied formatting from the one before.

Reply:
Ah ok, fair enough! I just checked and I think it also looks ok without the `<br>` tags but this is out of the scope of this PR.

Reply:
I removed it (: